solidity/contracts/Staking.sol

```diff
@@ -229,7 +243,9 @@ contract Staking {
         uint256 interest = calculateInterest(msg.sender);
         Staker storage stakerData = interestData.stakers[msg.sender];
         stakerData.withdrawnToDate = stakerData.withdrawnToDate.add(interest);
-        require(rewardToken.transfer(msg.sender, interest), "Withdraw interest transfer failed");
+        for (uint r = 0; r < RewardsTokens.length; r++) {
+            require(RewardsTokens[r].token.transfer(msg.sender, interest), "Withdraw interest transfer failed");
```

Comment:
This means you send the same amount of tokens without taking into account that total reward per token is differ